### PR TITLE
Math Magician: Events

### PR DESCRIPTION
**Pull request by Ghiftee**
This is the third milestone in the Math Magician app project. In this milestone, I used the already provided calculate.js and operate.js files to add functionalities to my calculator. These functionalities includes event handlers required to use the math logic.
The app can now perform math operations.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import calculate from '../logic/calculate';
+
```
Comment (DeliceLydia) [inline src/components/Calculator.jsx]: - [x] The Calculator component should import the `calculate.js & operate` js files from the logic directory, currently, you imported `calculate.js` only please fix that and all inconveniences importing can cause

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Ghiftee👏👏, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I recommend deleting all remained boilerplates like `reportWebVitals` or `setupTests.js` as you will not use them here


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.jsx (hunk 1 above)
Comment (Ghiftee) [reply src/components/Calculator.jsx]: I have imported the operate file from my logic directory into the components directory

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @Ghiftee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**